index.rst

```diff
@@ -29,4 +29,5 @@ For completing the tasks itself, however, we mainly rely on the work of the inte
    snakemake.md
    conda.md
    scripts/modify_files.rst
-
+   JupyterLab with version control <https://bamresearch.github.io/jupyter-integration/en_jupyterlab_with_version_control.html>
+   Using Git and TortoiseGit <https://bamresearch.github.io/jupyter-integration/en_version_control_with_git.html>
```

Comment:
Thank you for contribution and all that effort to setup the website for your jupyter-integration repo! And sorry for my delay, my notifications were messed up. 

I think @Psirus does not like that external links are within our "Table of Contents", because they are literally external and not the contents of this repo/website. And I would agree. Following his suggestions of simply pointing out that those are external links, I have the following suggestion:

```suggestion
From `BAM/jupyter-integration <https://github.com/BAMResearch/jupyter-integration>`_:
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

* `JupyterLab with version control <https://bamresearch.github.io/jupyter-integration/en_jupyterlab_with_version_control.html>`_
* `Using Git and TortoiseGit <https://bamresearch.github.io/jupyter-integration/en_version_control_with_git.html>`_
```

@ibressler Is that alright for you?

Reply:
yep, fine for me, added to the pull request.